### ISSUE TITLE
jpackage Packaging Tool

### ISSUE THREAD

**Issue by Kore-Development** [SKCraft/Launcher#423]
The methods of distributing the bootstrapper may be able to be replaced with jpackage. As far as I know, jpackage will not be an incubating feature in Java 17 meaning that *in production* you will likely need Java 17 to use jpackage. It became an incubating feature in Java 14.
https://github.com/SKCraft/Launcher/wiki/Configuring-Self-Update#distributing-executables

More info here:
https://www.baeldung.com/java14-jpackage

**Comment by Kore-Development** [comment]
To make it a simple EXE file instead of a folder that you would need to unzip, you could create an SFX archive with WinRAR.

**Comment by hedgehog1029** [comment]
Yes, jpackage and jmods are an interesting feature that might help with app distribution, since it allows you to bundle the required bits of the JRE with the app and create installable app images. I am looking into this already but it will probably take a while to actually be implemented.
